### PR TITLE
Changelog for #956

### PR DESCRIPTION
**Pull request by ndhanushkodi**
- [x] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ IMPROVEMENTS:
 * Helm
   * Allow customization of `terminationGracePeriodSeconds` on the ingress gateways. [[GH-947](https://github.com/hashicorp/consul-k8s/pull/947)]
   * Support `ui.dashboardURLTemplates.service` value for setting [dashboard URL templates](https://www.consul.io/docs/agent/options#ui_config_dashboard_url_templates_service). [[GH-937](https://github.com/hashicorp/consul-k8s/pull/937)]
+* Control Plane
+  * Allow managing consul sidecar resources via annotations. [[GH-956](https://github.com/hashicorp/consul-k8s/pull/956)]
```
Comment (t-eckert) [inline CHANGELOG.md:12]:
```suggestion
  * Allow managing Consul sidecar resources via annotations. [[GH-956](https://github.com/hashicorp/consul-k8s/pull/956)]
```

**Review by t-eckert — APPROVED** [review]
Manage those sidecar resources!

**Review by ishustava — COMMENTED** [review]
Thank you! Do we need to also update consul.io docs?

**Review by ishustava — APPROVED** [review]

**Review by kschoche — APPROVED** [review]
🥇

**Review by thisisnotashwin — APPROVED** [review]
I wanted to feel like i was at the approval party too.

**Comment by rrondeau** [comment]
I made this https://github.com/hashicorp/consul-k8s/pull/986 for the chart

**Comment by ndhanushkodi** [comment]
> Thank you! Do we need to also update consul.io docs?

@ishustava nice catch, thanks! yeah I'll make a docs pr

**Pull request merged**